### ISSUE TITLE
curl 7.48.0 (stable) vs libressl 2.2.6 (stable)

### ISSUE THREAD

**Issue by hdatma** [curl/curl#726]
### I did this
Update from curl 7.47.1 to curl 7.48.0

### I expected the following
no compilation errors

### curl/libcurl version
as above

### operating system
Irrelevant

### compilation error

vtls/openssl.c:739:27: error: too few arguments to function call, single argument 'tid' was not specified
  ERR_remove_thread_state();
  ~~~~~~~~~~~~~~~~~~~~~~~ ^
/opt/libressl/include/openssl/err.h:351:1: note: 'ERR_remove_thread_state' declared here
void ERR_remove_thread_state(const CRYPTO_THREADID *tid);
^
1 error generated.
make[2]: *** [vtls/libcurl_la-openssl.lo] Error 1
make[1]: *** [all] Error 2
make: *** [all-recursive] Error 1


### comments

Please make sure the stable release does not clash with 
the stable release of libressl, before actually releasing it.

**Comment by bagder** [comment]
Thanks, this is already fixed in 240cd84b494e (and then slightly cleaned up in 23ab481644)

**Comment by gvanem** [comment]
LibreSSL was not on my mind when creating the https://github.com/curl/curl/issues/713 issue.
